api/v1/helpers/verbs/utils.js

```diff
@@ -680,7 +681,7 @@ function forbidden(next, modelName) {
 // ----------------------------------------------------------------------------
 
 module.exports = {
-
+  logAPI,
```

Comment:
Not sure if this a good practice. Files not requiring logAPI will be importing the logAPI code for no reason.